twitter_delete.rb

```diff
@@ -19,12 +20,14 @@
   opt :force, "Actually delete/unfavourite/unretweet tweets"
   opt :user, "The Twitter username to purge", type: :string, default: ENV["TWITTER_USER"]
   opt :csv, "Twitter archive tweets.csv file", type: :string
+  opt :csvonly, "Will only delete tweets contained in the given csv file", type: :boolean
```

Comment:
In my case I deleted the tweets that I want to keep from the `tweets.csv` by hand